bundles/org.openhab.binding.konnected/README.md

```diff
@@ -1,13 +1,13 @@
 # Konnected Binding
 
-This binding is for interacting with the [Konnected Alarm Panel](https://konnected.io/).
-A module which interfaces with existing home security sensors.
-Konnected is an open-source firmware and software that runs on a NodeMCU ESP8266 (Wi-Fi) or ESP32 (Pro) device.
-The Konnected hardware is designed for an alarm panel installation, but the general purpose firmware/software can be run on a generic NodeMCU device.
+This binding is for interacting with an [Konnected Alarm Panel](https://konnected.io/). Konnected Alarm Panels can connect to 
```

Comment:
Please reformat the changes to have one sentence per line. This will be rendered the same way, but will provide better diffs.

Reply:
Done